package.json

```diff
@@ -7,8 +7,8 @@
     "index.js"
   ],
   "scripts": {
-    "test": "NODE_ENV=test mocha --check-leaks -R spec -t 5000 test/*.test.js",
-    "test-cov": "NODE_ENV=test istanbul cover _mocha -- --check-leaks -t 5000 test/*.test.js",
+    "test": "NODE_ENV=test mocha --exit --check-leaks -R spec -t 5000 test/*.test.js",
```

Comment:
`--exit` is required due to [this change in mocha](https://boneskull.com/mocha-v4-nears-release/#mochawontforceexit)